### PR TITLE
expose channel vote amounts

### PR DESCRIPTION
**Pull request by maikelmclauflin**
allows publishers to see pending channel votes

### CODE REVIEW

**Comment by nvonpentz** [comment]
Is the idea here to have publishers hit both /v1/accounts/balances and /v1/accounts/balances/pending, sum the result, then display the sum to the user?

**Comment by maikelmclauflin** [comment]
@nvonpentz either that, or i can move it under the transactions for it to sum as you had described.

**Comment by maikelmclauflin** [comment]
yeah, pretty sure this can happen

**Comment by evq** [comment]
perhaps we only change the `/accounts/balances` endpoint conditionally via a `pending=true` query parameter?  and forego changes to other APIs to keep things simple. I think in practice pending balances should be small so any discrepancy is unlikely to cause confusion. wdyt @maikelmclauflin / @nvonpentz

**Comment by nvonpentz** [comment]
That sounds good to me.  Publishers will always supply `pending=true` for both displaying balances and creating payout reports.

**Comment by maikelmclauflin** [comment]
yeah, that's fine

File: eyeshade/controllers/accounts.js [hunk 1]
```diff
@@ -84,17 +109,30 @@ ORDER BY created_at
...
     if (!Array.isArray(accounts)) {
       accounts = [accounts]
     }
+    const args = [accounts]
 
-    const query1 = `select * from account_balances where account_id = any($1::text[])`
-
-    const transactions = await runtime.postgres.query(query1, [ accounts ])
-    reply(transactions.rows)
+    const votesPromise = pending ? runtime.postgres.query(selectPendingAccountVotes, args) : {
```
Comment (evq) [inline eyeshade/controllers/accounts.js]: can we short circuit this for non-channel accounts?

**Comment by maikelmclauflin** [comment]
seems like travis is freezing up will try again in a couple of hours

**Pull request merged**